document/js-api/index.bs

```diff
@@ -1361,10 +1363,14 @@ For any [=associated store=] |store|, the result of
 
 To <dfn for=WebAssembly>throw</dfn> with a [=tag address=] |type|, a matching [=list=] of WebAssembly values |payload|, and an [=externref=] |opaqueData|, perform the following steps:
 
+1. Associate |opaqueData| with the identity of the thrown exception.
 1. Unwind the stack until reaching the *catching try block* given |type|.
```

Comment:
Indeed. Ideally this text should probably say something like "create an exception and unwind the stack" according to the language of the core spec, and do whatever the core spec says should be done with exceptions; i.e. unwind until the exception is caught or propagate it back to the embedder. Is there a particular spot in the core spec that defines that creation?

Reply:
Maybe what we actually need to do is define some kind of interface in the [embedding appendix](https://github.com/WebAssembly/exception-handling/blob/main/document/core/appendix/embedding.rst) for that. The question still remains though, since that interface will itself need to say what happens.

Reply:
https://github.com/WebAssembly/exception-handling/pull/268 is the beginning of what I meant by the above comment; although I noticed that it doesn't say anything about calls from the instance into the embedder, so there may not be anything relevant to the case we're talking about here.